### PR TITLE
Support rich menu

### PR DESCRIPTION
**Pull request by d-mato**
Add some methods for Rich Menu API.

- Line::Bot::Clinet#get_richmenus
- Line::Bot::Clinet#get_richmenu
- Line::Bot::Clinet#create_richmenu
- Line::Bot::Clinet#delete_richmenu
- Line::Bot::Clinet#link_user_richmenu
- Line::Bot::Clinet#unlink_user_richmenu
- Line::Bot::Clinet#get_richmenu_image
- Line::Bot::Clinet#create_richmenu_image

### CODE REVIEW

**Comment by hirohisa** [comment]
Thank you for your PR, @d-mato .

It would be good for us if variable name is `rich_menu` instead of `richmenu`.

File: lib/line/bot/request.rb [hunk 1]
```diff
@@ -29,8 +30,10 @@ def initialize
         yield(self) if block_given?
       end
 
-      # @return [Hash]
+      # @return [String]
```
Comment (hirohisa) [inline lib/line/bot/request.rb]: Thank you, I mistake.

File: lib/line/bot/request.rb [hunk 2]
```diff
@@ -42,8 +45,15 @@ def payload
 
       # @return [Hash]
       def header
+        content_type =
+          if file.is_a? File
+            MIME::Types.type_for(file.path).first.to_s
```
Comment (hirohisa) [inline lib/line/bot/request.rb]: It would be better `'application/json;` instead because reducing the effect of other modules.

**Comment by d-mato** [comment]
Sorry for late reply!
I created tests for some methods i implemented.
I'm waiting for your review.

File: lib/line/bot/request.rb (hunk 2 above)
Comment (d-mato) [reply lib/line/bot/request.rb]: Thank you for your advice.
Remove 'mime-types' https://github.com/line/line-bot-sdk-ruby/pull/65/commits/d4a3bcb9c9c8f2f8f2162dab37f47d869047c289

**Comment by hirohisa** [comment]
Thank you for this pull request.

**Pull request merged**